$OpenBSD$

Help files uses some old textproc system last released
over a decade ago.

--- configure.in.orig	Sun Jun  9 09:05:11 2013
+++ configure.in	Sat Sep 27 22:36:21 2014
@@ -8,10 +8,6 @@ AC_CONFIG_SRCDIR([src/hearts.c])
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
 
-# Scrollkeeper for the documentation
-SCROLLKEEPER_BUILD_REQUIRED=0.3.5
-AC_SUBST(SCROLLKEEPER_BUILD_REQUIRED)
-
 # Find a C compiler and some i18n tools
 AC_PROG_CC
 AC_LANG_C
@@ -31,10 +27,7 @@ AC_SUBST(GNOME_HEARTS_LIBS)
 AM_PATH_PYTHON(2.4)
 
 # Find Python compiler and linker flags
-PYTHON_LIBS=[$($PYTHON -c "from distutils import sysconfig
-import os
-print os.path.join(*sysconfig.get_config_vars('LIBDIR', 'LDLIBRARYDIR', 'LDLIBRARY')),
-sysconfig.get_config_var('LIBS'), sysconfig.get_config_var('LINKFORSHARED')")]
+PYTHON_LIBS=-lpython2.7
 PYTHON_CFLAGS=[$($PYTHON -c "from distutils import sysconfig
 print sysconfig.get_python_inc()")]
 PYTHON_CFLAGS="-I$PYTHON_CFLAGS"
@@ -60,12 +53,6 @@ AC_SUBST(PYTHON_LIBS)
 AC_SUBST(PYTHON_DIR)
 AC_DEFINE_UNQUOTED(PYTHON_DIR, "$PYTHON_DIR", [Python directory])
 
-
-# Check for scrollkeeper
-AC_PATH_PROG(SCROLLKEEPER_CONFIG, scrollkeeper-config,no)
-if test x$SCROLLKEEPER_CONFIG = xno; then
-  AC_MSG_ERROR(Couldn't find scrollkeeper-config. Please install the scrollkeeper package: http://scrollkeeper.sourceforge.net)
-fi
 
 # Set up gettext support
 GETTEXT_PACKAGE=gnome-hearts
